eth/sync.go

```diff
@@ -189,7 +189,7 @@ func (cs *chainSyncer) nextSyncOp() *chainSyncOp {
 		// We seem to be in sync according to the legacy rules. In the merge
 		// world, it can also mean we're stuck on the merge block, waiting for
 		// a beacon client. In the latter case, notify the user.
-		if cs.handler.chain.Config().TerminalTotalDifficulty != nil && time.Since(cs.warned) > 10*time.Second {
+		if ttd := cs.handler.chain.Config().TerminalTotalDifficulty; ttd != nil && ourTD.Cmp(ttd) >= 0 && time.Since(cs.warned) > 10*time.Second {
```

Comment:
Please revert. The point of this branch was to keep bugging the user until they start the beacon client. Your change just silently hangs without giving any iead to the user.

Reply:
Hmm, I misunderstood the code. I guess I forgot to check if we're past TTD and kept warning even if we're below but in-sync?

Reply:
I think so, don't really remember why I changed this